VideoLocker/src/main/java/org/edx/mobile/view/Router.java

```diff
@@ -329,4 +330,12 @@ public void showHandouts(Activity activity, EnrolledCoursesResponse courseData)
         handoutIntent.addFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT);
         activity.startActivity(handoutIntent);
     }
+
+    public void showUserProfile(@NonNull Context context, @NonNull String username) {
+        context.startActivity(UserProfileActivity.newIntent(context, username));
```

Comment:
Should we be delegating `Intent` creation to the `Activity` class? It's not the pattern for the rest of the methods…